nf_core/lint/multiqc_config.py

```diff
@@ -75,9 +75,9 @@ def multiqc_config(self):
             if "report_comment" not in mqc_yml:
                 raise AssertionError()
             if mqc_yml["report_comment"].strip() != (
-                f'This report has been generated by the <a href="https://github.com/nf-core/{self.pipeline_name}/{version}" '
+                f'This report has been generated by the <a href="https://github.com/nf-core/{self.pipeline_name}/tree/{version}" '
```

Comment:
Comment on line 74:
What about doing a regex split to check if the 'manifest.version' contains the 'dev' tag? Then it would also be possible to refer to the `dev` version.
Like 
```
import re
if (re.split("\d", version)[-1]=='dev'):
    version = 'dev'
```

Reply:
good idea, thanks for the suggestion! I have pushed the changes :)